batch2/batch/driver/templates/user_resources.html

```diff
@@ -0,0 +1,28 @@
+{% extends "layout.html" %}
+{% block title %}Batch User Resources{% endblock %}
+{% block content %}
+
+<h1>Users</h1>
+<table class="data-table" id="user_resources">
+    <thead>
+    <tr>
+        <th>Name</th>
+        <th>Ready Jobs</th>
+        <th>Ready Cores</th>
+        <th>Running Jobs</th>
+        <th>Running Cores</th>
+    </tr>
+    </thead>
+    <tbody>
+    {% for user in user_resources %}
+    <tr>
+        <td>{{ user['user'] }}</td>
+        <td class="numeric-cell">{{ user['n_ready_jobs'] }}</td>
```

Comment:
What I'd really like to see is the current ready/running, but also the fair share.  Can you call compute_fair_share and display that?